falcon/vendor/mimeparse/mimeparse.py

```diff
@@ -77,8 +86,8 @@ def quality_and_fitness_parsed(mime_type, parsed_ranges):
 
     :rtype: (float,int)
     """
-    best_fitness = -1
-    best_fit_q = 0
+    best_fitness = -1.0
+    best_fit_q = 0.0
```

Comment:
i've recently become aware that float is a supertype of int for type checkers, so we can do
```py
    best_fitness: float = -1
    best_fit_q: float = 0

Reply:
Changed.